src/routes/collections.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+
+import { getCollections } from '../controllers/collections.js';
+
+/**
+ * @swagger
+ * tags:
+ *   name: Collections
+ *   description: Collections API
+ */
+const router = express.Router();
+
+/**
+ * @swagger
+ * /collections:
+ *   get:
+ *     summary: Enumerate all collections.
```

Comment:
At this stage, if I understand the implementation correctly, there is no config choice at all and one has to rely on the list curated by the core team. Let's make that clear.
```suggestion
 *     summary: List all collections federated by the Open Terms Archive core team.
```